### ISSUE TITLE
Is current version can change the batch_size to a more bigger number?

### ISSUE THREAD

**Issue by hugddygff** [JaywongWang/DenseVideoCaptioning#10]
And can you share the training detaild of the pretrained proposal model you provided ? The batch_size is set to 1? 

Thanks very much!

**Comment by JaywongWang** [comment]
The batch size is set to 64 for the pre-trained proposal model. Current code does not support batch size > 1. You may need to decouple the code and change batch size to 64 for training  proposal model.

**Comment by hugddygff** [comment]
@JaywongWang Did you try beam search or model ensembling?

**Comment by JaywongWang** [comment]
> @JaywongWang Did you try beam search or model ensembling?

We tried beam search and found it did not improve meteor score. Suitable model ensembling will definitely work better. We did not use ensembling in our paper, just for fair comparison.

**Comment by hugddygff** [comment]
@JaywongWang Can you give a hint to decouple the code of proposal and caption, or, which part of the code/function need to be changed? Thanks a lot!

**Comment by JaywongWang** [comment]
Keep only codes for the proposal and remove the codes for the captioning part.  Please also refer to my implementation of SST: https://github.com/JaywongWang/SST-Tensorflow

**Comment by hugddygff** [comment]
@JaywongWang  I have decoupled the code successfully, and I want to ask the training details (experimental) of the proposal module? Thanks.

**Comment by hugddygff** [comment]
such as batch_size, learning rate, ....

**Comment by JaywongWang** [comment]
batch_size=32, learning_rate=1e-3, solver='adam', rnn_size=512

**Comment by hugddygff** [comment]
How about the number of epoch?

**Comment by JaywongWang** [comment]
Around 30 to 50 epochs. Depends on the loss or performance.

**Comment by JaywongWang** [comment]
Close issue.
